docs/streams.md

```diff
@@ -98,105 +90,94 @@ example, if you had the following events in a stream :
 
-`$persistentSubscriptionConfig` is a special paged stream that contains all configuration events, for all
-persistent subscriptions. It uses the `PersistentConfig` system event type, which records a configuration
-event. The event data contains:
+**`$persistentSubscriptionConfig`** is a specialized paged stream that stores all configuration events for all
+persistent subscriptions. It uses the **`PersistentConfig`** system event type, capturing configuration changes. 
+The event data includes:
 
-- `version`: Version of event data
-- `updated`: Updated date
-- `updatedBy`: User who updated configuration
-- `maxCount`: The number of configuration events to save
-- `entries`: Configuration items set by event.
+- **`version`**: Event data version
+- **`updated`**: Date of the update
+- **`updatedBy`**: User responsible for the configuration update
+- **`maxCount`**: The number of configuration events to retain
+- **`entries`**: Configuration items set by the event.
 
-### `$all`
+### **`$all`**
 
-`$all` is a special paged stream for all events. You can use the same paged form of reading described above to
-read all events for a node by pointing the stream at _/streams/\$all_. As it's a stream like any other, you
-can perform all operations, except posting to it.
+**`$all`** is a dedicated paged stream containing all events. You can employ the same paged reading method described earlier to
+read all events for a node by pointing the stream at _/streams/\$all_. Like any other stream, you can perform all operations, except posting to it.#
 
-### `$settings`
+### **`$settings`**
 
-The `$settings` stream has a special ACL used as the default ACL. This stream controls the default ACL for
-streams without an ACL and also controls who can create streams in the system.
+The **`$settings`** stream hosts a special ACL serving as the default ACL. It governs the default ACL for streams lacking ACLs and controls who can create streams within the system.
 
 Learn more about the default ACL in the [access control lists](security.md#default-acl) documentation.
 
-### `$stats`
+### **`$stats`**
 
-EventStoreDB has debug and statistics information available about a cluster in the `$stats` stream, find out
+EventStoreDB offers debugging and statistical information about a cluster within the **`$stats`** stream. Find out
 more in [the stats guide](diagnostics.md#statistics).
 
-### `$scavenges`
+### **`$scavenges`**
 
-`$scavenges` is a special paged stream for all scavenge related events. It uses the following system event
+**`$scavenges`** represents a specialized paged stream for all scavenge-related events. It uses the following system event
 types:
 
-- `$scavengeIndexInitialized`: An event that records the initialisation of the scavenge index.
-- `$scavengeStarted`: An event that records the beginning of a scavenge event, the event data contains:
+- **`$scavengeIndexInitialized`**: An event that records the initialization of the scavenge index.
```

Comment:
Again, I recommend a table. For those values with sub-values, we can use, for example `$scavengeCompleted.scavengeId`

Reply:
Actually I have a better idea on how to format this rather than a table, I'm gonna look into it. But leaving it as is for this PR.